.travis.yml

```diff
@@ -8,8 +8,6 @@ language: python
 matrix:
   fast_finish: true
   include:
-    - python: 3.4
```

Comment:
I removed this because 3.4 is not supported by Pytest 5. I can probably work out how to add logic if you would prefer me to use a different pytest version, but it seems like a lot of work for an ageing Python Version.

Reply:
The only reason is that the default version of Python on our RHEL7 machines is Python 3.4. But the reality is that we'll never use this library with that version of Python. So, I guess it is fair to drop 3.4.